bson/bsoncodec/struct_codec.go

```diff
@@ -74,6 +74,14 @@ func (sc *StructCodec) EncodeValue(r EncodeContext, vw bsonrw.ValueWriter, val r
 			rv = val.FieldByIndex(desc.inline)
 		}
 
+		if desc.encoder == nil && (rv.Kind() == reflect.Interface) {
```

Comment:
Is there a reason we can't use `lookupElementEncoder` here?

Reply:
done